modules/ROOT/partials/deployment/services/env-and-yaml.adoc

```diff
@@ -27,58 +27,33 @@ To exclude a deprecation file if not wanted - like when the file will just not e
 
 ////
 
-// read the xxx_deprecation.adoc file from ocis to evaluate deprecations but only if 'no_deprecation' is not set
-// see the deprecation evaluation in the next block how it gets handled 
-
-ifndef::no_deprecation[]
-// deprecation test tab 1, uncomment for testing
-include::{ocis-services-raw-url}{service_tab_1}{ocis-services-final-path}adoc/{service_name}_deprecation.adoc[]
-ifeval::["{show-deprecation}" == "true"]
-:has_deprecation_tab_1: true
-endif::[]
-
-ifndef::no_second_tab[]
-// deprecation test tab 2, uncomment for testing
-include::{ocis-services-raw-url}{service_tab_2}{ocis-services-final-path}adoc/{service_name}_deprecation.adoc[]
-ifeval::["{show-deprecation}" == "true"]
-:has_deprecation_tab_2: true
-endif::[]
-
-ifndef::no_third_tab[]
-// deprecation test tab 3, uncomment for testing
-include::{ocis-services-raw-url}{service_tab_3}{ocis-services-final-path}adoc/{service_name}_deprecation.adoc[]
-ifeval::["{show-deprecation}" == "true"]
-:has_deprecation_tab_3: true
-endif::[]
-endif::no_third_tab[]
-endif::no_second_tab[]
-
 ////
-if one tab deprecation is set to true, set it generally. ifdef with comma uses or (with + uses and).
-this will show the deprecation table in *all* tabs if one is true, independent if that tab has one or not.
-this is because attributes are only evaluated OUTSIDE a tab definition.
+important: the xxx_deprecation.adoc does not extist for extended or global envvars. it may change for global envvars! therefore we need to query 'no_deprecation' to skip importing - or if we exclicitely dont want to show deprecated content.
```

Comment:
```suggestion
Important: The xxx_deprecation.adoc does not exist for extended or global envvars. It may change for global envvars! Therefore we need to query 'no_deprecation' to skip importing - or if we explicitly don't want to show deprecated content.
```